app/assets/stylesheets/base.scss.erb

```diff
@@ -213,37 +205,52 @@
   }
 
   .icon {
-    position: absolute;
-    z-index: 1;
     padding: 0;
     height: 48px;
-    width: 100%;
+    margin-right: 10px;
 
     .metacodeImage {
       cursor: move;
-      position: relative;
-      left: -23px;
-      top: 1px;
-      width: 46px;
-      height: 46px;
-      background-size:46px 46px;
+      position: absolute;
+      left: -18px;
+      top: 6px;
+      width: 36px;
+      height: 36px;
+      background-size:36px 36px;
```

Comment:
some of this could turn into variables, but only if it makes sense. I'd like to see less and less magic numbers in css and more stuff based on proportions of other stuff.... but only if it's helpful. so if using a variable doesn't make sense here, don't do it, but I'm commenting so you can consider it